link/src/main/java/com/stripe/android/link/LinkActivity.kt

```diff
@@ -100,6 +101,10 @@ internal class LinkActivity : ComponentActivity() {
                     },
                     modifier = Modifier.fillMaxHeight(),
                     sheetState = sheetState,
+                    sheetShape = MaterialTheme.shapes.large.copy(
+                        bottomStart = CornerSize(0.dp),
+                        bottomEnd = CornerSize(0.dp)
+                    ),
```

Comment:
Minor drive-by tweak to remove the bottom corner radiuses from the bottom sheet.